dist/index.js

```diff
@@ -51,6 +52,13 @@ var _H = 'H'.charCodeAt(0);
 var _Y = 'Y'.charCodeAt(0);
 var _S = 's'.charCodeAt(0);
 
+function changeDpiBuffer(buffer, dpi, type) {
+  var headerChunk = new Uint8Array(buffer.slice(0, 33));
+  var rest = buffer.slice(33);
+  var changedHeader = changeDpiOnArray(headerChunk, dpi, type);
+  return Buffer.concat([changedHeader, rest]);
+}
```

Comment:
What happen when this gets used in a browsers? can we stop the code reaching this point if Buffer is not supported? Can we add  as a peer dependency some module that would allow to use this function in the browser too?

Reply:
Updated this to be browser/node agnostic.